packages/cli/src/deployment.ts

```diff
@@ -317,8 +317,9 @@ function createDeployer<Settings = unknown>(
     params: DeployContractParams<P>,
     taskTag?: string
   ): Promise<DeployContractResult<T>> => {
+    const isDevnet = network.networkId !== 0 && network.networkId !== 1
```

Comment:
Maybe we use `networkId % 3 == 2` following the Scala code. `%3 = 0, 1, 2` would be `mainnet, testnset, and devnet`.

Reply:
As discussed, let's keep this for now until the next network upgrade